src/styles/moyo-header.scss

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.logo__link {
+  display: flex;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  background: url("#home");
+}
+
+.nav {
+  display: flex;
+  background-color: #fff;
+  text-transform: uppercase;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav__item a {
```

Comment:
as in the html, use a class to target nav links, you don't want to be dependent on a specific tag used. We hardly ever style things by using tag selector.